packages/eslint-plugin/src/configs/core.js

```diff
@@ -12,9 +12,11 @@ const config = {
     'plugin:import/typescript',
     // Extended configs are applied in order, so these configs that turn other rules off should come last
     'prettier',
+    // Browser compatibility linter rules
+    'plugin:compat/recommended',
```

Comment:
discussed offline - this should be applied only for actively developed framework ( v9 ) -> thus we need to move this to `base.js`